src/tile.js

```diff
@@ -79,58 +65,34 @@ class Tile extends React.Component {
     // Stylistic changes after modal window opens
   }
 
+  //close modal window
   closeRecipe() {
     this.setState({
       recipeOpen: false,
     });
   }
 
-  ingredList(ingred, meas) {
-    var listItems = ""
-
-    for(let i = 0; i < ingred.length; i++) {
-      listItems = listItems + " " + ingred[i] + " " + "(" + meas[i] + ")" + ","
-    }
-
-    return listItems
-  }
-
   render() {//button composed of image, the name of recipe, then link
-    const flip = this.state.flipped;
 
     return (
       <div>
-        <button className="tile-back" onClick={this.openRecipe}>
-        View Detailed Info
+        <button className="styles.tile_back" onClick={this.openRecipe}>
```

Comment:
tile_back isn't defined in tile.css did you mean tile_front? also probably remove styles because of above import styles review comment